src/node.cc

```diff
@@ -1,4 +1,4 @@
-#include "node.h"
+﻿#include "node.h"
```

Comment:
Does this change add or remove that non-printing character? If it adds it, please drop this.